Services/Movies/Movies.API/Models/FilterLimits.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Movies.API.Models;
+
+public class FilterLimits
+{
+    public uint YearMin { get; set; }
+    public uint YearMax { get; set; }
+    public int LengthMin { get; set; }
+    public int LengthMax { get; set; }
```

Comment:
zasto nije sve uint?

Reply:
nije nam takav entitet :D

Reply:
e svaka nam cast kako smo to napravili XD